src/pytorch_lightning/utilities/model_summary/model_summary.py

```diff
@@ -254,6 +255,18 @@ def summarize(self) -> Dict[str, LayerSummary]:
 
         return summary
 
+    def summarize_parameter(self) -> List[Tuple[str, int]]:
+        """Summarizes the number of parameters in the parameter that is specified as a parameter explicitly."""
+        summary: List[Tuple[str, int]] = []
+        for (name, p) in self._model.named_parameters():
+            if any(s in name for s in [".weight", ".bias"]):
+                continue
```

Comment:
Why does it need to skip a parameter if its name is in `[".weight", ".bias"]`? Also, I think assuming that parameter names are either of `[".weight", ".bias"]` doesn't look reasonable as they can basically be any strings, no?